src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -446,6 +446,87 @@ paths:
         connectionId: '${vpc_connection_id}'
         httpMethod: POST
 
+  #
+  # Get an Anonymous Elector Document (AED) from its application identifier
+  # --------------------------------------------------------------------------------
+  '/eros/{eroId}/anonymous-elector-documents/applications/{applicationId}':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application.
+        schema:
+          type: string
+        in: path
+        required: true
+      - name: applicationId
+        description: The identifier of the Anonymous Elector Document.
```

Comment:
Is that right, or is it the ID of the AED's originating application (aka `sourceReference`)?

Reply:
You're right, it is the sourceReference, and will be found using `findByGssCodeInAndSourceTypeAndSourceReference`.  

Agree, wording is wrong, the Certificates & Temporary Certificates use the following wording:
```
The identifier of the Voter Card Application.
```

So maybe, this one should be:
```
The identifier of the Anonymous Voter Card Application.
```